pkg/driver/driver_openstack.go

```diff
@@ -143,8 +146,50 @@ func (d *OpenStackDriver) Create() (string, string, error) {
 	var podNetworkIds = make(map[string]struct{})
 
 	if len(networkID) > 0 {
-		serverNetworks = append(serverNetworks, servers.Network{UUID: networkID})
+		// network port dealing if subnetID is specified
+
+		klog.V(3).Infof("existing network %s specified, need to pre-allocate ports ", networkID)
```

Comment:
I think the comment here is misplaced. Shouldn't it be inside the `if` below.

Reply:
Changed the comment to make more sense and put it _above_ the `if`-statement.